ui/app/styles/components/info-table-row.scss

```diff
@@ -25,6 +25,14 @@
     textarea {
       min-height: 35px;
     }
+
+    .helper-text {
+      font-weight: normal;
+    }
+
+    .has-align-self-center {
+      align-self: center;
```

Comment:
what's this for? does it break anything in the existing places where we use info-table-row?

Reply:
It does.  As discussed, I made this so it affected all columns in this component.  We will see this small change in all areas that the table is used.  

![align-self-center](https://user-images.githubusercontent.com/6618863/78606200-2b59a700-781a-11ea-828d-c09ee1a51a2c.gif)